content/huaweicloud-cpi.md

```diff
@@ -0,0 +1,142 @@
+This topic describes cloud properties for different resources created by the HuaweiCloud CPI.
+The Huaweicloud CPI is designed for user most familiar with the Huawei cloud concepts and it only used in [Huaweicloud](https://www.huaweicloud.com/).
+
+## AZs {: #azs }
+
+Schema for `cloud_properties` section:
+
+* **availability_zone** [String, required]: Availability zone to use for creating instances. Example: `cn-north-1b`.
+
+Example:
+
+```yaml
+azs:
+- name: z1
+  cloud_properties:
+    availability_zone: cn-north-1b
+```
+
+---
+## Networks {: #networks }
+
+Schema for `cloud_properties` section used by dynamic network or manual network subnet:
+
+* **subnet_id** [String, required]: Subnet ID belong to VPC and  the instance will be created.
+* **security_groups** [Array, optional]: Array of security group names or UUIDs to apply for all VMs that are placed on this subnet.
+
+Example of manual network:
+
+```yaml
+networks:
+- name: default
+  type: manual
+  subnets:
+  - range: 10.10.0.0/24
+    gateway: 10.10.0.1
+    cloud_properties:
+      subnet_id: 3c8632e2-d9ff-41b1-aa0c-d455557314a0
+      security_groups: [huaweicloud-security-group]
+```
+
+Example of dynamic network:
+
+```yaml
+networks:
+- name: default
+  type: dynamic
+  cloud_properties:
+    subnet_id: 3c8632e2-d9ff-41b1-aa0c-d455557314a0
+```
+
+
+---
+## VM Types / VM Extensions {: #resource-pools }
+
+Schema for `cloud_properties` section:
+
+* **instance_type** [String, required]: Type of the instance. Example: `s3.large.2`.
+* **availability_zone** [String, required]: Availability zone to use for creating instances. Example: `cn-north-1b`.
+* **security_groups** [Array, optional]: Array of security group names or UUIDs to apply for all VMs that are placed on this network. Defaults to security groups specified by `default_security_groups` in the global CPI settings unless security groups are specified on one of the VM networks. If security groups are specified on a resource pool and a network, the resource pool security groups takes precedence since CPI v34+. In older CPI versions prior v34, security groups can either be specified for a network or a resource pool. Security group UUIDs can be used since CPI v39+.
+* **key_name** [String, optional]: Key pair name. Defaults to key pair name specified by `default_key_name` in the global CPI settings. Example: `bosh`.
+
+Example of an `s3.large.2` instance:
+
+```yaml
+resource_pools:
+- name: default
+  network: default
+  stemcell:
+    name: bosh-huaweicloud-kvm-ubuntu-trusty-go_agent
+    version: latest
+  cloud_properties:
+    instance_type: s3.large.2
+    availability_zone: cn-north-1b
+```
+
+---
+## Global Configuration {: #global }
+
```

Comment:
Could you please provide a schema for this global configuration section? Similarly to how keys in the cloud_properties have descriptions, it would be nice to have descriptions in these keys as well.